screenpy_selenium/resolutions/is_invisible.py

```diff
@@ -0,0 +1,19 @@
+"""
+Matches against a visible WebElement.
+"""
+
+from screenpy.resolutions.base_resolution import BaseResolution
+
+from .custom_matchers import is_invisible_element
+
+
+class IsInvisible(BaseResolution):
+    """Match on a visible element.
```

Comment:
I think the two docstrings here probably mean to say "invisible".

Reply:
whoops!  fixed.